crates/cli-tools/CHANGELOG.md

```diff
@@ -12,4 +12,4 @@
 
 ## 0.1.0
 
-<!-- Increment to skip CHANGELOG.md test: 0 -->
+<!-- Increment to skip CHANGELOG.md test: 1 -->
```

Comment:
And here, since it's already a pre-release with a similar change, we don't duplicate it and instead just bump the skip counter to acknowledge that the changelog is up-to-date.